packages/main/src/CardHeader.js

```diff
@@ -241,15 +241,17 @@ class CardHeader extends UI5Element {
 	}
 
 	_headerClick(event) {
-		event.stopImmediatePropagation(); // prevents the native browser "click" event from firing
+		// prevents the native browser "click" event from firing
+		event.preventDefault();
```

Comment:
Can we move 

event.preventDefault();
event.stopImmediatePropagation();

inside the "if" statement?